### ISSUE TITLE
spinnaker-rosco Debian package in Spinnaker 1.34.0 is completely broken

### ISSUE THREAD

**Issue by ashleykleynhans** [spinnaker/spinnaker#6945]
### Issue Summary:

spinnaker-rosco Debian package in Spinnaker 1.34.0 is completely broken and prevents Spinnaker version 1.34.0 from being installed

### Cloud Provider(s):

* aws
* gcp

### Environment:

* Ubuntu 22.04 LTS
* EC2 standalone instance - no k8s
* Halyard 1.65.0

### Feature Area:

* rosco

#### Expected Behavior

All Spinnaker Debian packages should be upgraded to the ones specified in the BOM for Spinnaker version 1.34.0

#### Actual Behavior

spinnaker-rosco Debian package in the Spinnaker 1.34.0 release is broken - it keeps saying the dpkg database is locked by another process when its actually locked by the installation of spinnaker-rosco itself

I also get errors about awscli already being installed and to use --upgrade - this has never been an issue on previous Spinnaker upgrades.

### Steps to Reproduce:

1. Install Spinnaker 1.32.4
2. Upgrade to Spinnaler 1.34.0

This should be done using halyard on a single instance, not a k8s cluster.

### Additional Details:

```
dpkg: error: dpkg database lock was locked by another process with pid 5146
Note: removing the lock file is always wrong, and can end up damaging the
locked area and the entire system. See <https://wiki.debian.org/Teams/Dpkg/FAQ>.
dpkg: error processing package spinnaker-rosco (--configure):
 installed spinnaker-rosco package post-installation script subprocess returned error exit status 2
Errors were encountered while processing:
 spinnaker-rosco
E: Sub-process /usr/bin/dpkg returned an error code (1)
````

**Comment by ashleykleynhans** [comment]
Issue is caused by installing Debian packages within a Debian package, not sure why it is doing this.

This is in the `spinnaker-rosco.postinst` script:

```
install_aws_cli2() {
  if [ "$ARCH" = "amd64" ]; then
    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
    curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb"
  elif [ "$ARCH" = "arm64" ]; then
    curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"
    curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_arm64/session-manager-plugin.deb" -o "session-manager-plugin.deb"
  fi
  unzip awscliv2.zip
  ./aws/install
  rm -rf awscliv2.zip ./aws
  dpkg -i session-manager-plugin.deb
  rm session-manager-plugin.deb
}
```

The `spinnaker-rosco.postinst` files for Rosco in Spinnaker 1.32.4 and Spinnaker 1.33.0 do not have this bash function and therefore do not suffer the same issue.   I am not sure why this was added for Spinnaker 1.34.0 but should probably be removed since its completely breaking the Rosco installation.

**Comment by ashleykleynhans** [comment]
I can confirm that commenting out the execution of the `install_aws_cl2` bash function in `/var/lib/dpkg/info/spinnaker-rosco.postinst` was able to work-around the issue and the installation could complete.

**Comment by dbyron-sf** [comment]
I've released [version 1.34.1 of Spinnaker](https://spinnaker.io/changelogs/1.34.1-changelog/) with the fix for this.
